docs/reference/search/request/scroll.asciidoc

```diff
@@ -38,29 +38,29 @@ should keep the ``search context'' alive (see <<scroll-search-context>>), eg `?s
 
 [source,js]
 --------------------------------------------------
-curl -XGET 'localhost:9200/twitter/tweet/_search?scroll=1m' -d '
+GET /twitter/tweet/_search?scroll=1m
 {
     "query": {
         "match" : {
             "title" : "elasticsearch"
         }
     }
 }
-'
 --------------------------------------------------
+// CONSOLE
+// TEST[setup:twitter]
 
 The result from the above request includes a `_scroll_id`, which should
 be passed to the `scroll` API in order to retrieve the next batch of
 results.
 
 [source,js]
 --------------------------------------------------
-curl -XGET <1> 'localhost:9200/_search/scroll' <2> -d'
+GET <1> /_search/scroll <2> 
 {
     "scroll" : "1m", <3>
     "scroll_id" : "c2Nhbjs2OzM0NDg1ODpzRlBLc0FXNlNyNm5JWUc1" <4>
 }
-'
 --------------------------------------------------
 
```

Comment:
`// CONSOLE`? You can do a `// TEST[s/c2Nhbjs2OzM0NDg1ODpzRlBLc0FXNlNyNm5JWUc1/"$body.scroll_id"/]` or something like that?